acvm-repo/acvm/src/compiler/transformers/csat.rs

```diff
@@ -70,7 +70,7 @@ impl CSatTransformer {
     /// by creating intermediate variables as needed.
     /// Having the correct width means:
     /// - it has at most one multiplicative term
-    /// - it has at most 'width-1' linear combinations terms.
+    /// - it uses at most 'width-1' witness, to account for the new intermediate variable
```

Comment:
```suggestion
    /// - it uses at most 'width-1' witness linear combination terms, to account for the new intermediate variable
```
Should we still keep the note about linear combinations?